content/en/docs/zero-code/java/_index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Java zero-code instrumentation
+linkTitle: Java
+aliases:
+  - /docs/java/automatic_instrumentation
+  - /docs/languages/java/automatic_instrumentation
+# FIXME: ensure version update script targets this file too
+cascade:
+  vers:
+    instrumentation: 2.3.0
+    otel: 1.37.0
```

Comment:
```suggestion
    otel: 1.38.0
```

Reply:
We might need to fix some automation here as well post merge

Reply:
Yes, this is already a task marked in #4428.